app/controllers/integrations/buildkite_controller.rb

```diff
@@ -28,4 +28,9 @@ def build_param
   def message
     build_param[:message]
   end
+
+  def release_params
+    extra_params = Samson::Hooks.fire(:release_params, project, build_param)
```

Comment:
hooks sounds good ... method name can be release_params, but if they hook is only used in buildkite it should be named buildkite_release_params and not release_params to avoid confusion